package-lock.json

```diff
@@ -2,145 +2,16183 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
-    "@babel/runtime": {
+    "@babel/code-frame": {
```

Comment:
I'm not actually sure what this file does :D What resulted in this huge change to it and what are the implications of it?

Reply:
Hi, this was a result of setting up tests my side, subsequently this is not needed anymore on my end to please ignore and apologies for not removing this myself prior to committing